ckanext/report/templates/report/view.html

```diff
@@ -14,7 +14,7 @@ <h1>{{ report.title }}</h1>
           {{ _('Generated') }}: {{ h.report__render_datetime(report_date, '%d/%m/%Y %H:%M') }}
       </p>
       {% if c.userobj.sysadmin %}
-      <div class="panel panel-info" style="width=700px">
+      <div class="panel panel-info">
```

Comment:
maybe need another class so that style can be re-injected.

Reply:
As far as I've seen, the style isn't needed on 2.8, and gets in the way on 2.9.